README.md

```diff
@@ -97,7 +97,11 @@ Alternatively, install the development version of the package with (few
 users should need to do this):
 
 ``` r
-remotes::install_github("epiforecasts/EpiNow2")
+# check whether {pak} is installed
```

Comment:
This is out of date.

Reply:
I've fixed this here [595bf0c](https://github.com/epiforecasts/EpiNow2/pull/407/commits/595bf0c020c1fb2b02f321175694580836d05f6c).